### PR TITLE
Oops, allow files in opted-in third_party directories.

### PR DESCRIPTION
**Pull request by matanlurey**
Closes https://github.com/flutter/flutter/issues/136551.

### CODE REVIEW

**Comment by cyanglaz** [comment]
I tired this, didn't work, @jason-simmons found out that the below works. Unsure why
```
!txt/
!txt/**
```

**Comment by matanlurey** [comment]
Ah, `/*` as an opt-out is better than `*`. Looks good now locally.

**Review by cyanglaz — APPROVED** [review]
LGTM
Verified locally it works

File: third_party/.gitignore [hunk 1]
```diff
@@ -13,11 +13,10 @@
...
 !accessibility/
 !canvaskit/
-!ninja/
 !spring_animation/
 !test_shaders/
 !tonic/
 !txt/
 !web_locale_keymap/
-!web_test_fonts/
+!web_test_font/
```
Comment (jason-simmons) [inline third_party/.gitignore:21]: Looks like the removal of the `s` is a typo?

Comment (matanlurey) [reply third_party/.gitignore]: Yup! Nice catch, thanks.

**Review by jason-simmons — APPROVED** [review]

**Pull request merged**